pkg/sql/plan2/build_delete.go

```diff
@@ -38,7 +38,7 @@ func buildDelete(stmt *tree.Delete, ctx CompilerContext) (*Plan, error) {
 		return nil, errors.New(errno.SyntaxErrororAccessRuleViolation, "cannot find delete table")
 	}
 
-	// build the stmt of select
+	// build the projection of select
 	var projectExprs tree.SelectExprs
```

Comment:
seams projectExprs is an unused variable?
then buildProjectionFromExpr , buildProjectionFromExpr are unused function?

Reply:
projectExprs is used to check whether the primary key is a filter condition